src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -191,8 +190,8 @@ class CallActor(key: CallKey, locallyQualifiedInputs: CallInputs, backend: Backe
     )
 
     val message = executionResult match {
-      case SuccessfulExecution(outputs, returnCode, hash) =>
-        WorkflowActor.CallCompleted(key, outputs, returnCode, if (workflowDescriptor.writeToCache) Some(hash) else None)
+      case SuccessfulExecution(outputs, returnCode, hash, resultsClonedFrom) =>
+        WorkflowActor.CallCompleted(key, outputs, returnCode, if (workflowDescriptor.writeToCache) Option(hash) else None, resultsClonedFrom)
```

Comment:
Realizing it was already there, I'm a little :frowning: at the if backed right into the line there